pkg/handler/v2/etcdrestore/etcdrestore.go

```diff
@@ -51,6 +51,10 @@ func CreateEndpoint(userInfoGetter provider.UserInfoGetter, projectProvider prov
 			return nil, err
 		}
 
+		if !c.Spec.Features[kubermaticv1.ClusterFeatureEtcdLauncher] {
```

Comment:
Its ok to do it here. But we are moving everything to webhooks to support GitOps at one point, so it would be better to do this in a webhook

Reply:
Thanks! I will take check how I can do it there.

Reply:
@lsviben added webhooks.

Reply:
Spoke too soon. Testing this locally now.